helpers/remote.py

```diff
@@ -60,6 +61,22 @@ def modify_cmd(cmd: str) -> str:
     return cmd
 
 
+def numa_nodes_count(node) -> int:
+    cmd = f"lscpu | grep -oP 'NUMA node\(s\):\s*\K\d+'"
+    out = node.run_cmd(cmd)
+    return int(out[0].decode().strip("\n"))
+
+
+def threads_count(node) -> int:
```

Comment:
Let's move `numa_nodes_count`, `threads_count`, `modify_cmd` functions to `INode` as protect function... I don't know why these function should be open.

Reply:
I did it, with and renamed INode to ANone, seems these function is not in class, because the INode was designed as interface, however now it's abstract class.